src/js/yeldaWebchatInjector.js

```diff
@@ -502,6 +512,12 @@ class YeldaChat {
       data.shouldBeOpened = data.shouldBeOpened ? true : false
     }
 
+    // Check if the SHOW_BOT_PARAMETER(yshowbot) parameter exists in the url
+    if (!data.shouldBeOpened) {
+      // if exists then assign true to shouldBeOpened parameter
+      data.shouldBeOpened = this.isUrlParamExists(config.SHOW_BOT_PARAMETER)
+    }
+
```

Comment:
1. The card changed a little bit after the call with Thomas (sorry)
We will use a generic param yparam to pass different strings depending the use we want to have for this
https://trello.com/c/7P6rnE3M/1192-webchat-open-from-get

The idea is to be able to add new feature easily without asking too many changes to the client

2. The value of yparam should only affect shouldBeOpened if yparam=yshowbot

```js
    // Check if the SHOW_BOT_PARAMETER(yshowbot) parameter exists in the url
    if (!data.shouldBeOpened && this.doesUrlParamForceChatToBeOpened()) {
      data.shouldBeOpened = true
    }

```